### PR TITLE
View accel data on phone

### PR DESCRIPTION
**Pull request by bantinglu**

### CODE REVIEW

File: src/c/PebbleAccelerometer.c [hunk 1]
```diff
@@ -1,11 +1,63 @@
...
+
+typedef struct {
+  uint64_t    sync_set;
+  uint64_t    sync_vib;
+  uint64_t    sync_missed;
+} sync_stats_t;
+
+static sync_stats_t   syncStats = {0, 0, 0};
+
+static bool isBlocked = false;;
```
Comment (sdownie2) [inline src/c/PebbleAccelerometer.c]: double ;

File: src/c/PebbleAccelerometer.c [hunk 2]
```diff
@@ -96,8 +193,7 @@ static void deinit()
 int main(void)
 {
   APP_LOG(APP_LOG_LEVEL_INFO, "main: entry:  %s %s", __TIME__, __DATE__);
-  init();
-  capture();
+  init();;
```
Comment (sdownie2) [inline src/c/PebbleAccelerometer.c]: double ;

File: src/c/PebbleAccelerometer.c [hunk 3]
```diff
@@ -1,11 +1,63 @@
...
+}
+
+
+void in_dropped_handler(AppMessageResult reason, void *context) {
+  // incoming message dropped
+  APP_LOG(APP_LOG_LEVEL_ERROR, "Message dropped. Reason: %d", (int)reason);
+}
+
 static void main_window_load(Window *window)
 {
```
Comment (sdownie2) [inline src/c/PebbleAccelerometer.c:57]: so where are we putting braces? it seems to switch between the next line and the same line as the method name.  I don't care which is it as long as it is consistent

Comment (bantinglu) [reply src/c/PebbleAccelerometer.c:57]: Lets do next line for now, ill make the changes in this PR

**Review by sdownie2 — APPROVED** [review]

**Pull request merged**